### PR TITLE
Update fr_fr.json to 1.16

### PR DESCRIPTION
**Pull request by JoyBates**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 1]
```diff
@@ -665,23 +730,32 @@
...
 	"messages.securitycraft:blockpocket.alreadyAssembled": "La poche bloc a déjà été assemblé !",
 	"messages.securitycraft:blockpocket.invalidBlock.rotation": "Le pilier à la position %1$s est mal tourné !",
 	"messages.securitycraft:blockpocket.invalidBlock": "Type de bloc non valide \"%2$s\" à la position %1$s ! (Attendu : \"%3$s\")",
 	"messages.securitycraft:blockpocket.blockInWay": "\"%2$s\" bloque l'espace à la position %1$s !",
 	"messages.securitycraft:blockpocket.unowned": "Vous ne possédez pas le bloc \"%2$s\" à la position %1$s !",
+	"messages.securitycraft:blockpocket.notEnoughItems": "Vous n’avez pas assez d’objets pour assembler une poche bloc de cette taille !",
 	"messages.securitycraft:blockpocket.activated": "Poche bloc activaté !",
 	"messages.securitycraft:blockpocket.deactivated": "Poche bloc désactivaté !",
+	"messages.securitycraft:blockpocket.assembled": "Poche bloc assemblée !",
+	"messages.securitycraft:sc_help": "%s: %s + %s",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json:755]: There's no need for this if it's the same as in the English file.

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 2]
```diff
@@ -335,24 +390,30 @@
...
+	"block.securitycraft.reinforced_polished_blackstone_wall": "Muret en pierre noire renforcé",
+	"block.securitycraft.reinforced_polished_blackstone_brick_wall": "Muret en pierre noire taillée renforcé",
+	"block.securitycraft.secret_sign": "Panneau secret en chêne",
+	"block.securitycraft.secret_spruce_sign": "Panneau secret en sapin",
+	"block.securitycraft.secret_birch_sign": "Panneau secret en bouleau",
+	"block.securitycraft.secret_jungle_sign": "Panneau secret en acajou",
+	"block.securitycraft.secret_acacia_sign": "Panneau secret en acacia",
+	"block.securitycraft.secret_dark_oak_sign": "Panneau secret en chêne noir",
+	"item.securitycraft.secret_crimson_sign_item": "Panneau secret carmin",
+	"item.securitycraft.secret_warped_sign_item": "Panneau secret biscornu",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json:403]: Why are the items where the blocks are?

File: src/main/resources/assets/securitycraft/lang/fr_fr.json [hunk 3]
```diff
@@ -335,24 +390,30 @@
...
 	"block.securitycraft.reinforced_redstone_lamp": "Lampe à redstone renforcée",
-	"block.securitycraft.reinforced_grass_path": "Chemin d'herbe renforcé",
-	"block.securitycraft.reinforced_stone_button": "Bouton en pierre renforcé",
-	"block.securitycraft.reinforced_oak_button": "Bouton en bois de chêne renforcé",
-	"block.securitycraft.reinforced_spruce_button": "Bouton en bois de sapin renforcé",
-	"block.securitycraft.reinforced_birch_button": "Bouton en bois de bouleau renforcé",
-	"block.securitycraft.reinforced_jungle_button": "Bouton en bois d'acajou renforcé",
-	"block.securitycraft.reinforced_acacia_button": "Bouton en bois d'acacia renforcé",
-	"block.securitycraft.reinforced_dark_oak_button": "Bouton en bois de chêne noir renforcé",
-	"block.securitycraft.reinforced_lever": "Levier renforcé",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/fr_fr.json:355]: Why delete these and move them somewhere else? Just keep them where they are...

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
There are some things missing here - translations that were present before but have been modified to reflect new behavior/features. Check out en_us.json at https://github.com/Geforce132/SecurityCraft/compare/7097827a1e77f672260760c87fa8dc5a2f0f1604...ecbe1688ea362c748b36662de7135fa660374100 and compare and adapt to your changes, please.

And it is **very** inconvenient if you pull request a translation for an update _after_ said update has been released. To prevent this in the future, I would suggest you join the SecurityCraft Discord server. We'll give you the Translators role, and whenever we are close to releasing an update to the mod, we're pinging the role with the new and changed translations. That way we can get them in _before_ the release.

**Comment by JoyBates** [comment]
Because I did this at 1 am and pasted the same entry twice.

On Oct 23, 2020, at 6:08 AM, bl4ckscor3 <notifications@github.com> wrote:

﻿

@bl4ckscor3 commented on this pull request.

________________________________

In src/main/resources/assets/securitycraft/lang/fr_fr.json<https://github.com/Geforce132/SecurityCraft/pull/206#discussion_r510808852>:

> +     "item.securitycraft.secret_crimson_sign_item": "Panneau secret carmin",
+       "item.securitycraft.secret_warped_sign_item": "Panneau secret biscornu",


Why are the items where the blocks are?

**Comment by JoyBates** [comment]
I relocated them to the same position as in en_us.json, as it makes it easier to compare the files when the lines match up.

On Oct 23, 2020, at 6:16 AM, bl4ckscor3 <notifications@github.com> wrote:

﻿

@bl4ckscor3 commented on this pull request.

________________________________

In src/main/resources/assets/securitycraft/lang/fr_fr.json<https://github.com/Geforce132/SecurityCraft/pull/206#discussion_r510812456>:

> -     "block.securitycraft.reinforced_grass_path": "Chemin d'herbe renforcé",
-       "block.securitycraft.reinforced_stone_button": "Bouton en pierre renforcé",
-       "block.securitycraft.reinforced_oak_button": "Bouton en bois de chêne renforcé",
-       "block.securitycraft.reinforced_spruce_button": "Bouton en bois de sapin renforcé",
-       "block.securitycraft.reinforced_birch_button": "Bouton en bois de bouleau renforcé",
-       "block.securitycraft.reinforced_jungle_button": "Bouton en bois d'acajou renforcé",
-       "block.securitycraft.reinforced_acacia_button": "Bouton en bois d'acacia renforcé",
-       "block.securitycraft.reinforced_dark_oak_button": "Bouton en bois de chêne noir renforcé",
-       "block.securitycraft.reinforced_lever": "Levier renforcé",


Why delete these and move them somewhere else? Just keep them where they were...

**Comment by bl4ckscor3** [comment]
You need to compare the old en_us.json file with the new en_us.json file, and then adapt the fr_fr.json file. It doesn't matter in what order the entries in fr_fr.json are as you are not comparing with that.

**Comment by JoyBates** [comment]
> You need to compare the old en_us.json file with the new en_us.json file, and then adapt the fr_fr.json file. It doesn't matter in what order the entries in fr_fr.json are as you are not comparing with that.

Seems more efficient than what I’ve been doing (Thanks, Geforce XP).

**Comment by JoyBates** [comment]
Okay old friend, let me know if you find more issues/suggestions.

**Review by bl4ckscor3 — APPROVED** [review]

**Pull request merged**